src/components/OurCoreValue.css

```diff
@@ -0,0 +1,6 @@
+.site-main-inner-wrapper-core {
```

Comment:
This class breaks the existing structure of the sections, and therefore will make it more difficult to make the website responsive.

I recommend using the `.site-main-inner-wrapper` class name in `OurCoreValue.tsx` as in other sections, and just ditching this CSS file entirely.